doc/api/http.md

```diff
@@ -1013,9 +1013,13 @@ Limits maximum incoming headers count. If set to 0, no limit will be applied.
 ### server.setTimeout([msecs][, callback])
 <!-- YAML
 added: v0.9.12
+changes:
+  - version: REPLACEME
+    pr-url: TODO
+    description: The default timeout changed from 120s to no timeout.
 -->
 
-* `msecs` {number} **Default:** `120000` (2 minutes)
+* `msecs` {number} **Default:** no timeout.
```

Comment:
Setting the timeout to `0` means "no timeout", right? If so, then maybe it's best to say set the default to `0` rather than a non-number?

Reply:
👍